src/util.js

```diff
@@ -1545,7 +1545,14 @@ export const importText = (itemsRanked, inputText) => {
   if (numLines === 1) {
 
     const focusOffset = window.getSelection().focusOffset
-    const strippedText = strip(text)
+    let strippedText = strip(text)
+
+    if (itemsRanked[0].key !== '' && strippedText[0] !== ' ') {
```

Comment:
Instead of using `itemsRanked[0].key`, use `destKey`.

Reply:
Note that `strippedText[0] !== ' '` will always be `true` because `strippedText` has been trimmed, so you can remove it.